src/openai-auth.ts

```diff
@@ -132,6 +134,23 @@ export async function getOpenAIAuth({
         await page.waitForSelector('input[type="password"]', { visible: true })
         await page.type('input[type="password"]', password, { delay: 10 })
         submitP = () => page.keyboard.press('Enter')
+      } else if (isWindowsLogin) {
+        await page.click('button[data-provider="windowslive"]')
+        await page.waitForSelector('input[type="email"]')
+        await page.type('input[type="email"]', email, { delay: 10 })
+        await Promise.all([
+          page.waitForNavigation(),
+          await page.keyboard.press('Enter')
+        ])
+        await delay(1500)
+        await page.waitForSelector('input[type="password"]', { visible: true })
```

Comment:
```suggestion
        await page.waitForSelector('input[type="password"]', { visible: true })
```
Don't think the delay is necessary since we're already waiting for the selector here

Reply:
aye; sometimes delays are necessary, though it's good to try and avoid them if possible